pages/views/_includes/navigation/footer.njk

```diff
@@ -20,8 +20,8 @@
   <div class="footer-common-links-container">
     <div class="footer-common-links">
       <div class="footer-copyright">
-        ESL v{{ env.version }} © 2021 <a class="footer-link" href="https://exadel.com">Exadel</a>, Inc.
-        Code licenced under <a class="footer-link" href="https://github.com/exadel-inc/esl/blob/main/LICENSE">MIT</a>
+        ESL v{{ env.version }} © 2021 <a class="footer-link" href="https://exadel.com" target="_blank">Exadel</a>, Inc.
```

Comment:
Can we change the year automatically instead of manually?

Reply:
If I remember correctly copyright year is attached to the license and should not change when 2022 begins.

Reply:
yes, we can add it as a value and use on our page (as `version` in this case)